public/sass/components/_submenu.scss

```diff
@@ -30,114 +21,7 @@
   }
 }
 
-.variable-value-link {
```

Comment:
How confident are we that this scss is not being relied on elsewhere in the codebase? 🤔

Reply:
pretty confident. [i searched the entire grafana org for `variable-value-link`](https://github.com/search?q=org%3Agrafana+variable-value-link&type=code) and the only hit in code is from an [abandoned app plugin](https://github.com/grafana/kentik-app/blob/12ea573ccd5ea5886e220cd9f32abd0b7c485377/src/styles/kentik-main.scss#L612). 

there's no real way of knowing whether people are using this as a selector externally to override css. they might be. but i don't think we can account for that tbh, else we'll never be able to change a classname ever 😅